app-modules/contact/src/Filament/Resources/ContactResource/Pages/ManageContactAlerts.php

```diff
@@ -77,7 +77,7 @@ public static function getNavigationItems(array $urlParameters = []): array
         $ownerRecord = $urlParameters['record'];
 
         /** @var Contact $ownerRecord */
-        $alertsCount = Cache::tags('alert-count')
+        $alertsCount = Cache::tags('{{alert-count}}')
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.